admin_tests/common_auth/test_views.py

```diff
@@ -0,0 +1,63 @@
+from django.test import Client, TestCase
+from admin.common_auth.models import MyUser
+from django.core.urlresolvers import reverse
+from admin.common_auth.forms import CustomUserRegistrationForm
+
+
+class UserRegFormTestCase(TestCase):
+    def test_user_reg_form_success(self):
+        form_data = {
+            'email': 'example@example.com',
+            'first_name': 'Zak',
+            'last_name': 'K',
+            'password1': 'password',
+            'password2': 'password',
+        }
+        form = CustomUserRegistrationForm(form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_user_reg_form_failure(self):
+        # Every field is required, password length should be >= 5 (need Django 1.9 to use AUTH_PASSWORD_VALIDATORS)
+        form_data = {
+            'email': '',
+            'first_name': '',
+            'last_name': '',
+            'password1': 'pass',
+            'password2': 'pass',
+        }
+        form = CustomUserRegistrationForm(form_data)
+        self.assertFalse(form.is_valid())
```

Comment:
How about at least checking for the number of errors on the form, since otherwise it's not clear from this test which fields are failing since `is_valid` is so coarse.